homeassistant/components/sonos/speaker.py

```diff
@@ -178,7 +178,7 @@ async def async_setup_dispatchers(self, entry: ConfigEntry) -> None:
                 async_dispatcher_connect(
                     self.hass,
                     signal,
-                    target,
+                    target,  # type:ignore[arg-type]
```

Comment:
Add an annotation for `dispatch_pairs` instead. Then the `type: ignore` shouldn't be necessary.
```py
dispatch_pairs: tuple[tuple[str, Callable[..., Any]], ...] = (...)
```